pkg/management/component.go

```diff
@@ -1288,7 +1288,32 @@ func (c *component) genericSendOnboardingEmail(ctx context.Context, accessToken
 		return errorhandler.CreateBadRequestError(constants.MsgErrAlreadyOnboardedUser)
 	}
 
-	onboardingRedirectURI, err := c.onboardingModule.ComputeOnboardingRedirectURI(ctx, realmName, customerRealm, realmConf)
+	// get param context-key
```

Comment:
I feel we overcomplicate things by putting the context-key in paramKV and then searching in it. Do we need the context-key in paramKV in call SendOnboardingEmail ?